### DISCUSSION TITLE
ReadYou `0.9.12` release planning

### DISCUSSION THREAD

**Discussion by mbestavros** [ReadYouApp/ReadYou#572] [General]
Read You has seen a ton of developer activity recently (yay!) and several major new features have been recently merged, including:

- GReader API support (@Ashinch)
- A big Compose dependency bump (@JunkFood02)
- Tons of other small improvements

I don't think `0.9.12` should be released any time soon as I write this, but it might be helpful to have a dedicated place to discuss what are and aren't "must-have" features for the release when it eventually does happen.

**Comment by mbestavros** [comment]
Here are the issues I personally think MUST be resolved before we think about a new release:

https://github.com/Ashinch/ReadYou/issues/562
https://github.com/Ashinch/ReadYou/issues/561
https://github.com/Ashinch/ReadYou/issues/535

This is only considering the issues open at the moment - it's possible that other big bugs may surface that we need to iron out as well.

I would have added "incremental sync for GReader" to this list, but after trying out @Ashinch's patch from yesterday adding just that, it works pretty well!

**Comment by Ashinch** [comment]
Sure, we need to close the [0.9.12 milestone](https://github.com/Ashinch/ReadYou/milestone/1) before release.
